hologram_dashboard/README.md

```diff
@@ -0,0 +1,20 @@
+## Preparation
+    1.  clone the repository with "git clone 
```

Comment:
1. Clone the repository
    git clone https://github.com/googleinterns/sarahchen-intern-2020.git